core/src/main/kotlin/org/evomaster/core/problem/httpws/HttpWsCallResult.kt

```diff
@@ -126,4 +127,7 @@ abstract class HttpWsCallResult : EnterpriseActionResult {
      */
     fun setVulnerableForSSRF(on: Boolean) = addResultValue(VULNERABLE_SSRF, on.toString())
     fun getVulnerableForSSRF() : Boolean = getResultValue(VULNERABLE_SSRF)?.toBoolean() ?: false
+
+    fun setResponseTime(responseTime: Long?) = addResultValue(RESPONSE_TIME, responseTime.toString())
```

Comment:
why `Long?` as input? if it is null, this method should simply not be called. ie, input should be `Long`